.Lib9c.Tests/Model/Stake/ContractTest.cs

```diff
@@ -118,10 +118,16 @@ public class ContractTest
                 lockupInterval));
         }
 
+        [Fact]
+        public void Constructor_With_StakePolicySheet_Throw_ArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => new Contract((StakePolicySheet)null));
+        }
```

Comment:
Since Unity used by NineChronicles project supports C# 9, it should use `nullable` checker too 🤔 Of course, it is good!

Reply:
Does this mean that I should enable the nullable reference type in the Contract class, and make sure to use the StakePolicySheet (non nullable) type when using the above constructor?